design/sketch.webidl

```diff
@@ -413,23 +413,23 @@ enum GPUStencilOperation {
 };
 
 dictionary GPUStencilStateFaceDescriptor {
-    GPUCompareFunction compare;
-    GPUStencilOperation failOp;
-    GPUStencilOperation depthFailOp;
-    GPUStencilOperation passOp;
+    GPUCompareFunction compare = "less";
+    GPUStencilOperation failOp = "keep";
+    GPUStencilOperation depthFailOp = "keep";
+    GPUStencilOperation passOp = "keep";
 };
 
 dictionary GPUDepthStencilStateDescriptor {
     GPUTextureFormat format;
 
-    boolean depthWriteEnabled;
-    GPUCompareFunction depthCompare;
+    boolean depthWriteEnabled = false;
+    GPUCompareFunction depthCompare = "always";
 
     GPUStencilStateFaceDescriptor stencilFront;
     GPUStencilStateFaceDescriptor stencilBack;
```

Comment:
If an app doesn't specify these, do they get the default values above? Or will we have to do that manually in implementations?

Reply:
I guess these should say ` = {}`.

Reply:
Unfortunately doesn't seem to be valid idl, I tried it earlier. (https://w3c.github.io/webidl2.js/checker/)